### PR TITLE
Polish translation for Jak 1

### PR DESCRIPTION
**Pull request by rafalekkB**
Functional Polish translation for Jak 1 (Subtitles, Custom PC Port Menus)
The translation is still a WIP on Crowdin, but the translated lines work in-game.
Still need to figure out how to translate the Pause Menu strings.

### CODE REVIEW

**Comment by xTVaser** [comment]
> Still need to figure out how to translate the Pause Menu strings.
Look at an example such as hungarian which has added translations for the base strings - https://github.com/open-goal/jak-project/blob/master/game/assets/jak1/text/game_base_text_hu-HU.json

**Comment by rafalekkB** [comment]
> Look at an example such as hungarian which has added translations for the base strings
I can see that they translated the strings, but I can't find the original which should get decompiled from the game, I assume.
I batch searched all of the files and can't find it. And translating it back from Hungarian would probably result in bad translation.

**Comment by LuminarLight** [comment]
Hello, I made the Hungarian translation.

I got the base game strings by going to 'jak-project/decompiler_out/jak1/assets', and there was game_text.txt - this file is generated from your ISO as I understand, during the extraction process.

**Comment by rafalekkB** [comment]
> 

I got it, it's just not formatted the exact same way as yours, that's why I couldn't find it.

**Comment by LuminarLight** [comment]
If I recall correctly, when I made my translations, the format was the same. Some time ago the format became json, but the extracted game_text still has the old format.

File: game/assets/jak1/text/game_base_text_pl-PL.json [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  "010f": "POKAZUJ WSKAZÓWKI",
+  "0110": "WYŚRODKUJ EKRAN",
+  "0111": "WŁ.",
+  "0112": "WYŁ.",
+  "0113": "RUSZAJ PRZYCISKAMI KIERUNKOWYMI",
+  "0114": "ENGLISH",
+  "0115": "FRANÇAIS",
+  "0116": "DEUTSCH",
+  "0117": "ESPAÑOL",
+  "0118": "ITALIANIE",
```
Comment (ManDude) [inline game/assets/jak1/text/game_base_text_pl-PL.json:23]: Italiano.

File: game/assets/jak1/text/game_base_text_pl-PL.json [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  "091e": "WESPNIJ SIĘ NA KLIF",
+  "091f": "SKAŁA GEJZEROWA",
+  "0f00": "SONY COMPUTER ENTERTAINMENT",
+  "0f01": "AMERICA",
+  "0f02": "PRZEDSTAWIA",
+  "0f03": "GRĘ STUDIA",
+  "0f04": "NAUGHTY DOG",
+  "0f05": "",
+  "0f06": "JAK I DAXTER:",
+  "0f07": "DZIEDZICTWO PREKURSORÓW™",
```
[the remark this answers is not in the document]
Comment (rafalekkB) [reply game/assets/jak1/text/game_base_text_pl-PL.json]: My bad, I just translated everything top to bottom.

**Comment by xTVaser** [comment]
I converted the PR to draft, mark it ready for review when you are finished and you want it merged (which maybe it already is?)

**Comment by rafalekkB** [comment]
> I converted the PR to draft, mark it ready for review when you are finished and you want it merged (which maybe it already is?)

I'm currently at 50% translation progress of subtitles on Crowdin, finishing which will take a couple more days. I'll mark it ready when I have pushed the finished subtitles into the PR.

**Comment by xTVaser** [comment]
You don't have to do the changes in both places, if you do them in crowdin I can automatically add them into the repository.

**Comment by rafalekkB** [comment]
Sure, just wanted to finish the subtitle translation first, but I guess we can just go ahead and push the current version through

**Comment by rafalekkB** [comment]
Okay, I see that the underscore symbol is causing errors even though that's what the decomplier spits out. I'll get rid of these strings as these are associated with memory card which is not used in OpenGOAL.

**Comment by rafalekkB** [comment]
Should be fine now

**Comment by xTVaser** [comment]
For clarification, the underscore gets turned into a "wide space"

**Comment by ManDude** [comment]
The reason it's an underscore is because this large space is mapped to code `0x5F`, which is ASCII for the underscore. It's very likely they used an underscore to represent this character.

**Review by ManDude — APPROVED** [review]

**Comment by xTVaser** [comment]
ok to test

**Pull request merged**